build/helper/metadata_add_all.py

```diff
@@ -395,10 +401,173 @@ def add_all_enum_metadata(enums, config):
     _add_enum_codegen_method(enums, config)
     for e in enums:
         enums[e] = _add_enum_value_python_name(enums[e], config)
+        enums[e]['python_name'] = enums[e]['python_name'] if 'python_name' in enums[e] else e
 
     return enums
 
 
+def _need_func_note(nd, config):
```

Comment:
There's no docstring on any of these new methods.

This func_note refers to something that is documentation related, but that's not obvious in any way by context (this is in a file called metadata_add_all.py) nor by the name of the function.